validation/jinja2/validation/record_audio.html

```diff
@@ -0,0 +1,28 @@
+{% extends 'validation/_base.html' %}
+
+{% block content %}
+    <title>Simple Recorder.js demo with record, stop and pause</title>
+    <button id="recNewEntryButton" class="button button--neutral button--right">Record a new Entry</button>
+    <h2>Instructions</h2>
+    <p>Use this page to record or re-record entries in Plains Cree. Start by entering the transcription (in Cree)
+    and the translation (in English) for the word or sentence you are going to record. Try to spell the Cree word(s)
+        as correctly as possible, as these recordings will be added to an existing entry if one exists.</p>
+    <p>Next, click the "Record" button. It will likely ask you permission to use your microphone, please click "Allow". You are now recording, as indicated
+    by the word "Recording..." on the button. When you are finished, press "Stop". Listen to your recording. If you're happy
+    with how it sounds, click "Save" next to that recording. Please record yourself saying this word or phrase up to 3 times,
+    clicking save next to each good recording and ignoring any that you are unhappy with. To record a new word or phrase,
+    click the "Record a new Entry" button.</p>
+    <p>Please note that pressing the "Record a new Entry" button or reloading the page will cause all your current unsaved
+        recordings to disappear, so make sure you're done with them and have saved all the good ones before doing either of those actions.</p>
```

Comment:
If you want to later do one of those popups that says, “You have created recordings that you have not uploaded, do you want to continue?”, I think [`beforeunload`](https://developer.mozilla.org/en-US/docs/Web/API/Window/beforeunload_event) is the thing for that.

That also suggests having a *delete* button next to each recording which feels even more feature-creepy, so these are totally just suggestions for the future.

Please remember we’re all telling you all these things you can do with this because we’re *excited* about the *great new feature*!